docs/getting-started/android/screenshots.md

```diff
@@ -25,7 +25,9 @@ After _screengrab_ completes, it will show you summary of the screenshots you ca
 
 ## Getting Started Using Espresso
 
-_screengrab_ uses the capabilities of Android's built-in [Instrumented tests](https://developer.android.com/training/testing/unit-testing/instrumented-unit-tests.html) combined with [Espresso](https://developer.android.com/topic/libraries/testing-support-library/index.html#Espresso) to drive interactions with your app. To get familiar with writing UI Tests with Espresso, check out [Testing UI for a Single App](http://developer.android.com/training/testing/ui-testing/espresso-testing.html).
+_screengrab_ uses the capabilities of Android's built-in [Instrumented tests](https://developer.android.com/training/testing/unit-testing/instrumented-unit-tests.html) combined with [Espresso](https://developer.android.com/topic/libraries/testing-support-library/index.html#Espresso) to drive interactions with your app. 
+
+To get started with writing UI Tests with Espresso, [Create UI tests with Espresso Test Recorder](https://www.youtube.com/watch?v=JRkDVvB106k) video gives a quick intro to Espresso Recording feature for writing UI tests. For more details regrading UI testing, refer to the Anroid guide: [Testing UI for a Single App](http://developer.android.com/training/testing/ui-testing/espresso-testing.html).
```

Comment:
After `video gives a quick into to`
add `the` before `Espresso Recording feature`